### PR TITLE
A note on default requests

### PR DESCRIPTION
**Pull request by jnothman**
Subsequent to Dec 2020 meeting: https://github.com/scikit-learn/scikit-learn/pull/16079#issuecomment-747301842

### CODE REVIEW

**Comment by jnothman** [comment]
Sorry, this wording might not be great, @adrinjalali @agramfort  @lorentzenchr 

In terms of getting this SLEP to a votable state, would it be worth the effort to restructure it as "main proposal" and "alternatives", or do you think the current structure is accessible enough?

File: slep006/proposal.rst [hunk 1]
```diff
@@ -433,6 +433,15 @@ Having considered the above solutions, we propose:
   details of the solution.
 * Props will be known simply as Metadata.
 * `**kw` syntax will be used to pass props by key.
+* Default requests: A group splitter should return `['groups']` by default
+  because requiring `groups` is essential to their functionality. In other
```
Comment (adrinjalali) [inline slep006/proposal.rst:437]: Maybe also add: if a splitter can work with and w/o `groups`, it should not request `groups`.

**Review by adrinjalali — COMMENTED** [review]
I personally wouldn't be too picky, and when calling for a vote, I'd call for a vote on option 4, so it'd be still a binary vote.

**Comment by agramfort** [comment]
@jnothman I took your text in https://github.com/scikit-learn/enhancement_proposals/pull/50

I suggest we close this one and iterate on https://github.com/scikit-learn/enhancement_proposals/pull/50

**Pull request closed without merging**